src/scene/text/canvas/CanvasTextSystem.ts

```diff
@@ -78,48 +65,27 @@ export class CanvasTextSystem implements System
             options.style = new TextStyle(options.style);
         }
 
-        const { texture, canvasAndContext } = this.createTextureAndCanvas(
-            options as {text: string, style: TextStyle, resolution?: number}
-        );
-
-        this._renderer.texture.initSource(texture._source);
-
-        CanvasPool.returnCanvasAndContext(canvasAndContext);
-
-        return texture;
-    }
-
-    public createTextureAndCanvas(options: {text: string, style: TextStyle, resolution?: number})
```

Comment:
If this is a public API we should keep it around and call `CanvasTextGenerator` under the hood for compatibility